### DISCUSSION TITLE
Will this work with my ASUS STRIX G18

### DISCUSSION THREAD

**Discussion by 07Gking** [seerge/g-helper#1467] [General]
Hi, I just got a new STRIX G18 (G814JZ) and it's my first time setting up a gaming laptop. I found this usefull tool to replace the Preinstalled stuff. I haven't seen the G18 in the list of working devices. Thats why I'm quickly checking in to be 100% sure.
Thanks -G

**Comment by seerge** [comment]
It does work on all relatively-modern asus laptops
